### ISSUE TITLE
`--clear-screen --watch` slowely scrolls down after each rebuild

### ISSUE THREAD

**Issue by vlatkoB** [spacchetti/spago#465]
- spago v0.10.0.0
- linux, bash

When using this cmd (with `build` or `bundle-app`, and both directly or via `yarn`)
```bash
spago bundle-app --clear-screen --main Main --to dist/app.js --watch
```
The screen on the first run:
```
yarn run v1.19.1
$ spago bundle-app --clear-screen --main Main --to dist/app.js --watch
Installation complete.
           Src   Lib   All
Warnings   0     0     0  
Errors     0     0     0  
Build succeeded.
Bundle succeeded and output file to dist/app.js
Success! Waiting for next file change.
Type help for available commands. Press enter to force a rebuild.
```
The screen on the second run:
```
(empty line)
(empty line)
(empty line)
(empty line)
(empty line)
(empty line)
(empty line)
(empty line)
(empty line)
(empty line)
File changed, triggered a build: ".../Router.purs"
Compiling Main
           Src   Lib   All
Warnings   0     0     0  
Errors     0     0     0  
Build succeeded.
Bundle succeeded and output file to dist/app.js
Success! Waiting for the next file change.
Type help for available commands. Press enter to force a rebuild.
```
Next builds simply show below the previous one and so on.

**Comment by jhrcek** [comment]
I briefly looked into this and it seem it should be easy to fix. Just change this line 

https://github.com/spacchetti/spago/blob/0bdf296d14efaba9fb6798768c5281446dafe9a2/src/Spago/Watch.hs#L68

like this:

```diff
-          when (shouldClear == DoClear) $ liftIO clearScreen
+          when (shouldClear == DoClear) $ liftIO $ do
+            clearScreen
+            setCursorPosition 0 0
```

Tried this locally and it seems to work fine, but it only clears the screen on rebuild (after you change the file), not on the initial build. Would the expected behavior be to also clear the screen on the initial run of the command?

**Comment by vlatkoB** [comment]
Thanks for looking into it. It works as expected. :-)

I do not think the first build should trigger clearing the screen, so it seems to me it is working correctly.

Will you make a PR?

**Comment by jhrcek** [comment]
Sure, I will

**Comment by vlatkoB** [comment]
A question related to `--watch`. What about adding the `--ignore-pattern` (command line and/or config), so `--watch` does not jump on every file change?
With `emacs`, when you edit any file, it creates a `lockfile` named `.#edited_filename` in the same directory and `--watch` kicks in.

**Comment by f-f** [comment]
@vlatkoB I think we should just ignore the emacs pattern in general, as you'd always want to exclude these kinds of events.

I'll close this since it has been fixed in #466 and I opened #477 to track your usecase
